src/org/usfirst/frc/team4915/steamworks/OI.java

```diff
@@ -181,19 +180,24 @@ public Command getAutoCommand()
-        m_autoPresetOptions.put("Drive and Shoot Position 3", new GenericCommandGroup(m_robot.getDrivetrain(), this, 
-                35,135,24,Double.NaN,Double.NaN)); // This is the length from the diamond plate with the robot length and an inch (just to be safe) subtracted
-        m_autoPresetOptions.put("Drive, Shoot, and Cross Baseline Position 3", new GenericCommandGroup(m_robot.getDrivetrain(), this, 
-                35,135,24,-90,Double.NaN)); // This is the length from the diamond plate with the robot length and an inch (just to be safe) subtracted
-        
+        m_autoPresetOptions.put("Cross Baseline Positons 1+3", new DriveCommandGroup(m_robot.getDrivetrain(), this,
+                "Drive", "-95")); // This is the length from the diamond plate to the baseline
+        m_autoPresetOptions.put("Place Gear Position 2", new DriveCommandGroup(m_robot.getDrivetrain(), this,
+                "Drive", "" + (-(93.3 - (RobotMap.ROBOT_LENGTH + 1))))); // This is the length from the diamond plate with the robot length and an inch (just to be safe) subtracted
+        //         ^ There are too many parenthesis here, but to make it clear why we are using these number, we break everything up like this
+        m_autoPresetOptions.put("Drive and Shoot Position 1", new DriveCommandGroup(m_robot.getDrivetrain(), this,
+                "Drive", "35", "Turn", "90", "Drive", "" + (248 - (RobotMap.ROBOT_WIDTH / 2)), "Turn", "135", "Drive", "17")); // Drive out for the turning radius + 10 inches to be aligned with the middle of the boiler, drive the distance from the baseline minus the robot's width and then turn to be parallel with the boiler, and then drive into the boiler
+        m_autoPresetOptions.put("Drive and Shoot Position 2", new DriveCommandGroup(m_robot.getDrivetrain(), this,
+                "Drive", "35", "Turn", "90", "Drive", "" + (124 - (RobotMap.ROBOT_WIDTH / 2)), "Turn", "135", "Drive", "17")); // Drive out for the turning radius + 10 inches to be aligned with the middle of the boiler, drive the distance from the baseline minus half of the robot's width (we're centered on the baseline) and then turn so we're parallel with the boiler and drive into the boiler
+        m_autoPresetOptions.put("Drive and Shoot Position 3", new DriveCommandGroup(m_robot.getDrivetrain(), this,
+                "Drive", "35", "Turn", "135", "Drive", "24")); // This is the length from the diamond plate with the robot length and an inch (just to be safe) subtracted
+        m_autoPresetOptions.put("Five Feet", new DriveCommandGroup(m_robot.getDrivetrain(), this,
+                "Drive", "60")); // This is the length from the diamond plate with the robot length and an inch (just to be safe) subtracted
+        m_autoPresetOptions.put("Ten Feet", new DriveCommandGroup(m_robot.getDrivetrain(), this,
+                "Drive", "120")); // This is the length from the diamond plate with the robot length and an inch (just to be safe) subtracted
```

Comment:
I don't think your comments here (and in the commands above) match up with the numbers.

Also, inline math like that is impenetrable.  Maybe add some local variables above the `.put`s where you explain the movements a little more?